src/service/real-time-config/real-time-config-impl.js

```diff
@@ -502,45 +515,67 @@ export class RealTimeConfigManager {
     timeoutMillis,
     callout,
     checkStillCurrent,
-    errorReportingUrl
+    errorReportingUrl,
+    element
   ) {
+    let rtcFetch;
+    if (isAmpScriptUri(url)) {
+      rtcFetch = Services.scriptForDocOrNull(element)
+        .then((service) => {
+          userAssert(service, 'AMP-SCRIPT is not installed.');
+          return service.fetch(url);
```

Comment:
This is probably me not understanding how amp script works, but how to we ensure they are using the `no-dom` attr?

Reply:
From a security standpoint, I don't think we actually need it to be `nodom`. 
I.e. some pubs may want to have their amp-script perform both functions:
1. As a normal amp-script
2. As something that exports a fn for RTC

All the usual amp-script restrictions apply (re. dom mutation without user activity, etc)

Reply:
Going to followup on this to see if `sandboxed` should force nodom